src/components/pages/datasets/company/UKCompanyList.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+import Table from 'components/table/Table';
+import PageBox from 'components/pages/PageBox';
+import { useTranslation } from 'react-i18next';
+
+
+const UKCompanyList = () => {
+  const { t } = useTranslation();
+  const columns = [
+    { header: t('edrpou'), prop: 'edrpou' },
+    { header: t('name'), prop: 'name' },
+    { header: t('status'), prop: 'status' },
+    { header: t('address'), prop: 'address' },
+    { header: t('registrationDate'), prop: 'registration_date' },
+  ];
+  return (
+    <PageBox header={t('viewTheRegistry')} noBox>
+      <Table
+        columns={columns}
+        url="uk/"
```

Comment:
is this working url?

Reply:
yes, it works

Reply:
Everything works for me without errors, I rechecked it several times.